features/cli/cancel_unpaid_orders.feature

```diff
@@ -0,0 +1,25 @@
+@canceling_unpaid_orders @cli
```

Comment:
in almost all behats we have suite name in the first line but `context` (@api, @ui) of test we are putting above scenarios name. IMHO you should move it above line 16